strings/find.py

```diff
@@ -0,0 +1,25 @@
+# Find a string
+"""Task
+In this challenge, the user enters a string and a substring. You have to print the number of times that the substring
+occurs in the given string. String traversal will take place from left to right, not from right to left.
+
+NOTE: String letters are case-sensitive.
+"""
+
+
+def count_substring(string, sub_string):
+    count = 0
+    pos = string.find(sub_string)
+    for _ in range(0, len(string)):
+        while pos != -1:
+            pos = string.find(sub_string, pos + 1)
+            count += 1
+    return count
```

Comment:
If line12 returned `-1` which means no matches - why would we ever iterate for `len(string)` times in line 13? You can quit before it with zero matches

Reply:
I'll recast it